### DISCUSSION TITLE
AR隱形眼鏡成真、動動眼球就能拍照導航！原型機離量產更進一步

### DISCUSSION THREAD

**Discussion by Lin-NZ** [1101-GS3073-C/Python-Community-GS3073-C#727] [科技新知]
content :
https://3c.ltn.com.tw/news/48461
Reflections / Thoughts:
簡單來說，Mojo Vision的AR隱形眼鏡，本身就是一個世界最小的micro LED顯示器，嵌入了AR、相機、眼部追蹤等裝置，不用頭戴厚重的頭盔或眼鏡，眼睛看出去就是AR世界，動動眼球就能拍照、導航等，實現科幻電影裡的黑科技。
令人感到鼓舞的產品更新，希望再不久的未來能夠體驗到這種新型穿戴科技的便利

**Comment by Valentinaa0214** [comment]
頭戴式裝置精簡到做成隱形眼鏡真的是蠻厲害的，
不過隱形眼鏡畢竟是侵入性的，讓人難免有些害怕這種電子產品裝置會不會對人體帶來傷害，像是可能會過熱、電磁波之類的

**Comment by yjpxi** [comment]
長時間佩戴這類高科技隱形眼鏡，可能對眼睛健康造成未知的影響，持續使用微LED顯示和眼部追蹤可能會導致眼睛疲勞，甚至對視力產生負面影響，而且設備的複雜性，一旦出現故障，可能難以迅速修復或更換。
